src/main/kotlin/no/nav/personbruker/dittnav/api/config/httpClient.kt

```diff
@@ -53,11 +53,11 @@ suspend inline fun <reified T> HttpClient.getWithTokenx(url: URL, accessToken: A
     }
 }
 
-suspend inline fun <reified T> HttpClient.getExtendedTimeout(url: URL, user: AuthenticatedUser): T = withContext(Dispatchers.IO) {
+suspend inline fun <reified T> HttpClient.getWithMeldekortTokenx(url: URL, accessToken: AccessToken): T = withContext(Dispatchers.IO) {
     request {
         url(url)
         method = HttpMethod.Get
-        header(HttpHeaders.Authorization, user.createAuthenticationHeader())
```

Comment:
Er det navnet på TokenX-headeren som gjør at vi må ha en egen get-funksjon for meldekort?

Reply:
Ja. Slik som oss hadde de samme problem med at sidecar slukte alle tokens som kom inn på vanlig "Authorization", og definerte egen header for tokenx.